.env.development

```diff
@@ -3,5 +3,5 @@ ARANGO_USER=root
 ARANGO_PASSWORD=thereisnospoon
 ARANGO_ROOT_PASSWORD=thereisnospoon
 ARANGO_URL=http://localhost
-ARANGO_PORT=8529
+ARANGO_PORT=8530
```

Comment:
dev and test port are the same. Was this intentional?

Reply:
No, forgot to ask. What's a good practice? I didn't want them at the same port as prod. Is 3 different ports alright? Something like, prod at 8529 (arangodb default), dev at 8530 and test at 8531?